app/experimenter/experiments/api/v5/query.py

```diff
@@ -1,8 +1,53 @@
 import graphene
 
-from experimenter.experiments.api.v5.types import NimbusExperimentType
+from experimenter.experiments.api.v5.types import (
+    NimbusExperimentType,
+    NimbusFeatureConfigType,
+    NimbusLabelValueType,
+    NimbusProbeSetType,
+)
 from experimenter.experiments.constants.nimbus import NimbusConstants
-from experimenter.experiments.models.nimbus import NimbusExperiment
+from experimenter.experiments.models.nimbus import (
+    NimbusExperiment,
+    NimbusFeatureConfig,
+    NimbusProbeSet,
+)
+
+
+class NimbusConfigurationType(graphene.ObjectType):
+    application = graphene.List(NimbusLabelValueType)
+    channels = graphene.List(NimbusLabelValueType)
+    feature_config = graphene.List(NimbusFeatureConfigType)
+    firefox_min_version = graphene.List(NimbusLabelValueType)
+    probe_sets = graphene.List(NimbusProbeSetType)
+    targeting_config_slug = graphene.List(NimbusLabelValueType)
+
+    def _text_choices_to_label_value_list(root, text_choices):
+        return [
+            NimbusLabelValueType(
+                label=text_choices[name].label,
+                value=name,
```

Comment:
In the V5 mutation tests, it looks like we send the `value`, not the `name`:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/tests/api/v5/test_mutation.py#L142

Shouldn't we be sending the `value` out here then too?

Reply:
At line 395 in that file, we use the name. Apparently graphql doesn't care?

Reply:
I just tried passing in `NimbusExperiment.Application.DESKTOP.name` to the create mutation and got:

```js
{'data': {'createExperiment': {'nimbusExperiment': None, 'message': {'application': ['"DESKTOP" is not a valid choice.']}, 'status': 200, 'clientMutationId': 'randomid'}}}
```

Reply:
Ah looks like that field isn't validating using the enums correctly, we should file a followup